procurement_mrp_no_confirm/i18n/procurement_mrp_no_confirm.pot

```diff
@@ -0,0 +1,32 @@
+# Translation of Odoo Server.
```

Comment:
Yes, please, it's no longer needed as Transifex sync script generates it on each commit and can confuse users.